ui/jobs/jobs.js

```diff
@@ -453,8 +453,10 @@ class Bars {
       timerBox.id = id;
       timerBox.classList.add('timer-box');
     }
-    if (notifyWhenExpired)
+    if (notifyWhenExpired) {
       timerBox.classList.add('notify-when-expired');
+      timerBox.soundnotification = this.options.NotifyExpiredProcsInCombatSound;
```

Comment:
I think these options could be separate.  I think `notifyWhenExpired` could be just the animation and then `this.options.NotifyExpiredProcsInCombatSound;` could be just the sound, and then somebody could turn one on but not the other.

Reply:
Oh right, sorry.  `notifyWhenExpired` is whether the proc box itself is on an proc where this should happen at all.  Got it, sorry for the confusion!